Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -815,11 +815,11 @@ def panorama_get_address(address_name: str) -> Dict:
     return result['response']['result']['entry']
 
 
-def panorama_get_address_command(name: str):
+def panorama_get_address_command(args: dict):
     """
     Get an address
     """
-    address_name = name
+    address_name = args.get('name')
```

Comment:
same question as the tag

Reply:
If there will be no args passed to `panorama_list_addresses_command` in `**args`, the code will break. which is a plausible scenario in the TPB.